### PR TITLE
calendar

### PR DESCRIPTION
**Pull request by Max-just**
- [DEMO LINK](https://Max-just.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Max-just.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--mon1"></div>
```
Comment (maxim2310) [inline src/index.html]: you dont need this enywhere
```suggestion
      <div class="calendar__day"></div>
```

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```
Comment (maxim2310) [inline src/index.html]: you need to add modificators for calendar and write scss func for position and hide entire items
(must looks like calendar--start-day--wed calendar--month-length--30)

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good work, but need to improve it

File: src/index.html (hunk 2 above)
Comment (Max-just) [reply src/index.html]: but i dont know how to write func

**Review by Max-just — COMMENTED** [review]
- [DEMO LINK](https://Max-just.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Max-just.github.io/layout_calendar/report/html_report/)

**Review by Max-just — COMMENTED** [review]
- [DEMO LINK](https://Max-just.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Max-just.github.io/layout_calendar/report/html_report/)

File: src/index.html [hunk 3]
```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--30">
+      <div class="calendar__day mon-first"></div>
```
Comment (sTorba24) [inline src/index.html]:
```suggestion
    <div class="calendar calendar--month-length--30 calendar--start-day-mon">
      <div class="calendar__day"></div>
```

Comment (sTorba24) [reply src/index.html]: you should implement these 2 modifiers for `calendar` block

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Good work!

Finish with all requirements before requesting review

if you have any problems with it - text your fe_chat to solve it

**Review by Max-just — COMMENTED** [review]
- [DEMO LINK](https://Max-just.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Max-just.github.io/layout_calendar/report/html_report/)

**Review by lerastarynets — APPROVED** [review]
But pls make sure the number in modifier corresponds to the number of days shown in the calendar
Screenshot at Nov 10 18-45-13

**Review by Max-just — COMMENTED** [review]
- [DEMO LINK](https://Max-just.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Max-just.github.io/layout_calendar/report/html_report/)

**Pull request closed without merging**